examples/dbus-info/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": 0,
+
+    "tools": {
+        "timezone": {
+            "label": "Atomic",
+            "path": "atomicClient.html"
+        }
+    },
+
+    "content-security-policy": "default-src 'self' 'unsafe-inline' 'unsafe-eval'"
```

Comment:
Lets remove this line. We'll need to move the script into an external atomic-client.js file in order for this to work.